### ISSUE TITLE
Toggle chats (global / local)

### ISSUE THREAD

**Issue by Isolonice** [MultiChat/Development#3]
Hello,

Using lastest snapshot (9)
Global chat is default chat, even if we do /local it still using the global chat.

**Comment by OllieMartin** [comment]
Hey,
The way global chat and local chat work in multichat is rather odd at the moment. WHen you do /local it will still put your messages in the global chat, but the only difference is that you will only see messages from people who are on the same server as you are. I appreciate its a bit weird, and when I do the major chat stream overhaul for the plugin ready for version 2.0 I will make sure this is a much better system for using local chats etc. In the mean time if you want to force a server to use the built in local chat then you can exclude it from the global chat in the config file.

I might try and squeeze into the 1.4 update some code to make /local mean that only people on the same server as you can see YOUR messages as well as the other way around.

If you have any other issues let me know :)

**Comment by Isolonice** [comment]
Well that would be nice, because the global chat is usefull for networks with differents kind of servers.  If there is no local chat per servers they are all talking of different things (and asking). 

Something like:
- /local : toggle local chat, you send local messages
- /global : toggle global chat, you see local messages but you send global messages / If global was on, turns off so you won't see global chat anymore
/g <message> : So you can send global messages without toggling

If you toggle "/global" you can see global chat, if you toggle local after it you see all kind of messages but you'r sending to local.

**Comment by OllieMartin** [comment]
Yeah as Im sure you can see it all starts to get very complicated trying to figure out what chat you are looking at or typing into or what and then youll get people sending messages trying to figure out why others cant see them if you have local and global on and so many things like that lol

Its such a confusing aspect and I have been designing a way that I think will work best to have as many chat streams as you like and this will come with version 2 which will be the next major version after 1.4. But yeah Id like to add an extra bit of diversity in 1.4 for people like you who would benefit from these features :) So what I'll most like do is just make it so that /global means you see the global chat and speak into global chat, but if you are on the same server as people in local chat the message will also go to them. Then if you are in local chat you will only see messages from people on the same server regardless of if they are local or global, and your messages will only go to people on the same server as you are. I know its not the most ideal scenario but I think its the best one which I can easily push out with 1.4 and its definitely better than the current /local! 2.0 Will definitely help you out and youll be able to define all kinds of chat streams and such to have different combinations of servers etc. and an easy way for users to decide where they are chatting and what streams they are viewing etc.

Hope this helps! :)

**Comment by Isolonice** [comment]
It will ! (help :D) Thanks ;)

**Comment by OllieMartin** [comment]
So I decided why not and did it just now. 30 minutes is quite impressive!

https://github.com/MultiChat/Development/blob/master/multichat/Versions/1.4/1.4.0/Snapshots/MultiChat%201.4%20Snapshop%2010.jar

**Comment by OllieMartin** [comment]
MultiChat 1.4 has now been released :)

**Comment by Isolonice** [comment]
Excellent ! Thanks !
